workers/unity/Packages/com.improbable.gdk.health/Systems/HealthRegenSystem.cs

```diff
@@ -131,75 +126,63 @@ private void ApplyHealthRegen()
                 return;
             }
 
-            var spatialIdData = regenGroup.GetComponentDataArray<SpatialEntityId>();
-            var healthComponentData = regenGroup.GetComponentDataArray<HealthComponent.Component>();
-            var healthRegenComponentData = regenGroup.GetComponentDataArray<HealthRegenComponent.Component>();
-            var healthRegenData = regenGroup.GetComponentDataArray<HealthRegenData>();
-
-            // Count down the timers, and update the HealthComponent accordingly.
-            for (var i = 0; i < spatialIdData.Length; i++)
-            {
-                var healthComponent = healthComponentData[i];
-                var regenComponent = healthRegenComponentData[i];
-                var regenData = healthRegenData[i];
-
-                // Don't regen if dead.
-                if (healthComponent.Health == 0)
+            Entities.With(regenGroup).ForEach(
+                (ref SpatialEntityId spatialEntityId,
```

Comment:
nit: indenting of first one is different

Reply:
This was done by our formatting rules